containers/podman-desktop
# fix: e2e test

### What does this PR do?

Wait for elements inside the page and check dashboard is displayed
- take screenshots into `<root-folder>/tests/output/screenshots` folder
- record video into `<root-folder>/tests/output/videos` folder

Logs browser script into the stdout and redirect devtools console to the stdout.

ensure `NODE_ENV=development` is used to resolve correctly the `import.meta.env.*` correctly for tests

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/2312
fixes https://github.com/containers/podman-desktop/issues/1780

### How to test this PR?

Unit test should be green

`yarn test:e2e`

ensure that settings are empty for now (so welcome page is displayed)
 on macOS: `rm ~/.local/share/containers/podman-desktop/configuration/settings.json`




Change-Id: I4b22e9fa21ce6fc6a85abefb7737c5ba8cbce173
